src/Components/PresentationalComponents/CVEPageDetailsDescription/CVEPageDetailsExpandable.js

```diff
@@ -0,0 +1,49 @@
+import { Text, TextContent } from '@patternfly/react-core';
+import propTypes from 'prop-types';
+import React, { useState, useEffect } from 'react';
+import './CVEPageDetailsDescription.scss';
+import { DESCRIPTION_CHARACTER_LIMIT } from '../../../Helpers/constants';
+
+const CVEPageDetailsExpandable = props => {
```

Comment:
what if this component is not used only for the `CVEPageDetail` page? How would you call it?

Reply:
Should I wrap the props in { }, or do you mean something else?